README.md

```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install django==1.7
```

Comment:
It's django's [bug](https://code.djangoproject.com/query?owner=nobody&desc=1&order=id&row=description)